api/src/cli/index.ts

```diff
@@ -92,7 +92,7 @@ export async function createCli(): Promise<Command> {
 		.description('Create a new Schema Snapshot')
 		.option('-y, --yes', `Assume "yes" as answer to all prompts and run non-interactively`, false)
 		.addOption(new Option('--format <format>', 'JSON or YAML format').choices(['json', 'yaml']).default('yaml'))
-		.addOption(new Option('--split <split>', 'Export as a single schema file').choices(['yes', 'no']).default('no'))
+		.addOption('-s, --split', 'Export as a single schema file', false)
```

Comment:
Would it possible to update this description? It feels like it's describing the opposite from my current interpretation, but I could be misunderstanding it.

Reply:
Ah right, you are right